features/core/src/main/java/org/apache/karaf/features/internal/service/FeatureConfigInstaller.java

```diff
@@ -252,12 +252,16 @@ private void installConfigurationFile(String fileLocation, String finalname, boo
         finalname = substFinalName(finalname);
 
         File file = new File(finalname);
+        if (!file.getCanonicalPath().startsWith(System.getProperty("karaf.base"))) {
```

Comment:
I'm -1 about this as some users have `karaf.etc` "outside" of `karaf.base`. I would replace with:

```
if (!file.getCanonicalPath().startsWith(System.getProperty("karaf.etc")))
```

Reply:
Generally speaking, I don't want to be too much strict for users. Some users have "exploded" Karaf in several folders. This change is too restrictive to then IMHO.

Reply:
@jbonofre  I've fixed it to use karaf.etc now, please review again.